### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by sanieni6**
## Portfolio: add a contact form and finish the mobile version
Changes done for the new sections on the portfolio project:
+ Colabore with my code parnter
+ Add the new contact form section
+ Add styles for the new contact form section
+ Add the new social network section in the footer
+ Add styles for the new social network section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,5 +184,42 @@ <h3 class="format">Skills</h3>
       </li>
     </ul>
   </section>
+  <section class="form">
+    <p class="primary-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <form action="https://formspree.io/f/mwkjplad" method="post">
+      <ul class="element-form">
+        <li class="items-form">
+          <!-- <label for="name"></label> -->
```
Comment (Owusu-Desmond) [inline index.html:192]: - [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

File: styles.css [hunk 2]
```diff
@@ -494,3 +495,130 @@ button:disabled {
...
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin: 0%;
+}
+
```
Comment (Owusu-Desmond) [inline styles.css:522]: - [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Your project is not responsive on a mobile screen at the screen width of `375px`; due to that, you have a horizontal scroll. Make sure that your project is responsive and has the design the as the Figma design at a mobile screen width of `375px`
 
| Your Design |Figma Design |
| :---:   | :---: | 
| Screenshot from 2023-01-17 17-28-57| Screenshot from 2023-01-17 17-36-46|

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @sanieni6 and @Gardimy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @sanieni6, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**